identity-service/src/config.js

```diff
@@ -106,6 +106,12 @@ const config = convict({
     env: 'relayerWallets',
     default: null
   },
+  ethFunderAddress: {
```

Comment:
this is a NIT but i think we should specify the use case for new configs here in the descriptions - these are used _only_ in `balance_check` and `eth_balance_check`

Reply:
basically these configs have no impact on any functionality other than the status code returned from those routes